src/Http/Routing/test/UnitTests/Patterns/RoutePatternFactoryTest.cs

```diff
@@ -218,6 +218,67 @@ public void Pattern_ExtraConstraints()
                 });
         }
 
+        [Fact]
+        public void Pattern_ExtraConstraints_MultipleConstraintsForKey()
```

Comment:
Please also add a test with strings, and mixed policies and strings. That's the thing that someone is most likely to "clean up" later.